See if running autoreconf lets us build shared libraries.

pkgw opened the pull request:
Yargh, Windows!

pkgw committed:
See if running autoreconf lets us build shared libraries.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
Need to specify versions of aclocal and autoconf.

pkgw committed:
Further work to get autoreconf going.

This is based on prototyping work done on a VM. Happily this work indicates
that autoreconf does indeed get us building DLL's instead of static libraries!

pkgw committed:
Wrong prefix for pkg-config aclocal file.

pkgw committed:
Fix autoreconf for Win32.

pkgw committed:
Update for new xorg-util-macros and try to fix the ws2_32 libtool issue

pkgw committed:
Diagnostic for package testing.

pkgw commented:
This PR has served its investigative purpose; soon to be superseded.
